packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1304,6 +1304,9 @@ abstract class ResidentRunner {
     }
     await waitForExtension(device.vmService, 'ext.flutter.activeDevToolsServerAddress');
     try {
+      if (_devToolsLauncher == null) {
+        return;
```

Comment:
Should this print an error like the one below in the catch block?

Reply:
no, this indicates that the app started closing before we finished this block. so perhaps the app crashed and we already printed a message?